src/visitpy/visit_utils/src/qannote/items.py

```diff
@@ -34,7 +41,7 @@ def process_encoded_text(val):
     while curr != -1:
         res += val[prev:curr]
         cval = int(rexp.cap(1)[1:],base=16)
-        res +=  unichr(cval)
+        res +=  py2to3_compat_unichr(cval)
```

Comment:
Any idea why this is the only place we appear to have to deal with unicode characters?

Reply:
this is special b/c of qt. In the past we were doing a conversion to unicode so we could have fancier text in qt, but unicode will be default in py3